README.md

```diff
@@ -47,12 +47,28 @@ for (var x = 0; x < 100; x++) {
     });
 }
 ```
+
+### Async
+Rapidly assigning network calls to be run, but they will be limited to 1 request per second and returns Promise.
+```js
+var throttledQueue = require('throttled-queue');
+var throttle = throttledQueue(1, 1000).asyncThrottle; // at most make 1 request every second.
```

Comment:
Not sure I fully follow, but could it be a flag instead? like `intermediate: true` ?